### DISCUSSION TITLE
reset option

### DISCUSSION THREAD

**Discussion by dev0pa** [iotaledger/firefly#1569] [Features / Enhancements]
Reset option when starting firefly (delete AppData folder)
Who has forgotten the password can not log in otherwise

**Comment by maxwellmattryan** [comment]
Hey @dev0pa this might not be a secure functionality to add. If the password to a profile is unknown, it is impossible to verify whether the user trying to delete the profile is the actual "owner" of that profile (i.e. an attacker might just try to delete the profile of a victim) in which case we do __not__ want the user to be able to delete a profile from the outside.

But as you mention the data (for Windows) is in `AppData`, so it is still possible to manually / forcefully delete the profile data anyway.
